docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+    "<font size=\"2\">[4] https://en.wikipedia.org/wiki/Amino_acid</font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bf140d37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "main_chain_len = 7\n",
+    "main_chain_residue_seq = [\"A\", \"P\", \"R\", \"L\", \"R\", \"F\", \"Y\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e399913f",
+   "metadata": {},
+   "source": [
+    "### Side Chains"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d71b4571",
+   "metadata": {},
+   "source": [
+    "Beyond the main chain of the protein there may be aminoacids attached to the residues of the main chain. Our model allows for side chains of the maximum length of one. Enlongated side chains would require the introduction of additional penalty terms that are still under development. \n",
+    "\n",
+    "For the definition of the side chain we provide a binary vector specifying the existence of a side chain and if a side chain is present we indicate the letter type of the residue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "4d148961",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "side_chain_lens = [0, 0, 0, 0, 0, 0, 0]\n",
+    "side_chain_residue_sequences = [None]*7"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "948d6e75",
+   "metadata": {},
+   "source": [
+    "### Interaction between Aminoacids"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "960e20e9",
+   "metadata": {},
+   "source": [
+    "For the description of inter-residue contacts for proteins we use knowledge-based (statistical) potentials derived using quasi-chemical approximation. The potentials used here are introduced by Miyazawa, S. and Jernigan, R. L. in [5]. \n",
+    "\n",
+    "Beyond this model we also allow for random contact maps (interactions) that provide a random interaction map. One can also introduce a custom interaction map that enhances certain configurations of the protein (e.g. alpha helix, beta sheet etc). \n",
+    "\n",
+    "<font size=\"2\"> [5] J. Mol. Biol.256, 623–644 (1996), Table 3. </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "26908e43",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "random_interaction = RandomInteraction()\n",
+    "mj_interaction = MiyazawaJerniganInteraction()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94612caf",
+   "metadata": {},
+   "source": [
+    "### Physical Constrains"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf834eb2",
+   "metadata": {},
+   "source": [
+    "To ensure that all physical constrains are respected we introduce penalty functions. The different penalty terms used are: \n",
+    "- penalty_chiral: A penalty parameter used to impose the right chirality.\n",
+    "- penalty_back: A penalty parameter used to penalize turns along the same axis.\n",
```

Comment:
Reading this I don't see how `back` and `axis` are related.. Is this supposed to be the `"backbone"` of the peptide?